docs/nodes/mamaki-testnet.md

```diff
@@ -89,25 +89,10 @@ cp $HOME/networks/mamaki/genesis.json $HOME/.celestia-app/config
 Set seeds and peers:
 
 ```sh
-BOOTSTRAP_PEERS=$(curl -sL https://raw.githubusercontent.com/celestiaorg/networks/master/mamaki/bootstrap-peers.txt | tr -d '\n')
-echo $BOOTSTRAP_PEERS
-sed -i.bak -e "s/^bootstrap-peers =./bootstrap-peers = "$BOOTSTRAP_PEERS"/" $HOME/.celestia-app/config/config.toml
-PEERS="e4429e99609c8c009969b0eb73c973bff33712f9@141.94.73.39:43656\
-  09263a4168de6a2aaf7fef86669ddfe4e2d004f6@142.132.209.229:26656,\
-  13d8abce0ff9565ed223c5e4b9906160816ee8fa@94.62.146.145:36656,\
-  72b34325513863152269e781d9866d1ec4d6a93a@65.108.194.40:26676,\
-  322542cec82814d8903de2259b1d4d97026bcb75@51.178.133.224:26666,\
-  5273f0deefa5f9c2d0a3bbf70840bb44c65d835c@80.190.129.50:49656,\
-  7145da826bbf64f06aa4ad296b850fd697a211cc@176.57.189.212:26656,\
-  5a4c337189eed845f3ece17f88da0d94c7eb2f9c@209.126.84.147:26656,\
-  ec072065bd4c6126a5833c97c8eb2d4382db85be@88.99.249.251:26656,\
-  cd1524191300d6354d6a322ab0bca1d7c8ddfd01@95.216.223.149:26656,\
-  2fd76fae32f587eceb266dce19053b20fce4e846@207.154.220.138:26656,\
-  1d6a3c3d9ffc828b926f95592e15b1b59b5d8175@135.181.56.56:26656,\
-  fe2025284ad9517ee6e8b027024cf4ae17e320c9@198.244.164.11:26656,\
-  fcff172744c51684aaefc6fd3433eae275a2f31b@159.203.18.242:26656"
-sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers \
-    *=.*/persistent_peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
+BOOTSTRAP_PEERS=$(curl -sL https://raw.githubusercontent.com/celestiaorg/networks/master/mamaki/bootstrap-peers.txt | tr -d '\n' | head -c -1) && echo $BOOTSTRAP_PEERS
+sed -i.bak -e "s/^bootstrap-peers *=.*/bootstrap-peers = \"$BOOTSTRAP_PEERS\"/" $HOME/.celestia-app/config/config.toml
+PEERS=$(curl -sL https://raw.githubusercontent.com/celestiaorg/networks/master/mamaki/peers.txt | tr -d '\n' | head -c -1) && echo $PEERS
+sed -i.bak -e "s/^persistent-peers *=.*/persistent-peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
```

Comment:
Please avoid including orthogonal changes

Reply:
sorry i dont understand what you mean